saleor/graphql/invoice/mutations.py

```diff
@@ -35,11 +35,11 @@ class Arguments:
 
     @staticmethod
     def clean_order(order):
-        if order.status == OrderStatus.DRAFT:
+        if order.status in (OrderStatus.DRAFT, OrderStatus.UNCONFIRMED):
```

Comment:
Maybe we should create some const for this status? For example `UNCONFIRMED_ORDER_STATUS= [ ...`

Reply:
Draft is not unconfirmed so it might cause some confusion, also when moving to constant I would have to jump to that when looking at the code to know what's going on. Maybe let's leave it explicit for now, I'll be refactoring some of order confirmation to be compliant with new feature so if it'll be needed elsewhere i'll extract it.